Lib/glyphsLib/builder.py

```diff
@@ -76,7 +77,7 @@
     '': 5}
 
 
-def to_ufos(data, include_instances=False, family_name=None, debug=False):
+def to_ufos(data, include_instances=False, family_name=None, debug=False, glyph_data=None):
```

Comment:
This is wrong. Why are you setting `glyph_data` default to `None` here, if a value is required for `build_gdef` and `get_glyph` to be able to work? Setting it as None implies it's optional, but it isn't. That's why you also changed all the calls to `to_ufos` in the `builder_test` to explicitly (superfluously) pass a `glyph_data` argument... Maybe you wanted to set the default value to `glyph_data=glyphdata_generated`.